src/components/Slider/SliderItem.tsx

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable import/extensions */
+import { Container } from '..';
```

Comment:
import container from relative path not from exporting index
this circle call and not good practice